### PR TITLE
Hipcc bin fixup

### PR DESCRIPTION
**Pull request by david-salinas**

### CODE REVIEW

File: src/hipBin_amd.h [hunk 1]
```diff
@@ -222,7 +222,8 @@ void HipBinAmd::constructCompilerPath() {
       complierPath = getRoccmPath();
       hipClangPath = complierPath;
     }
-    if (fs::exists("llvm/bin/clang++")) {
+
+    if (fs::exists(complierPath + "llvm/bin/clang++")) {
```
Comment (emankov) [inline src/hipBin_amd.h:226]: How is the trailing slash/backslash of the `complierPath` taken into account?

File: src/hipBin_amd.h [hunk 2]
```diff
@@ -718,8 +719,14 @@ void HipBinAmd::executeHipCCCmd(vector<string> argv) {
       regex reg("[^-a-zA-Z0-9_=+,.\\/]");
       arg = regex_replace(arg, reg, "\\$&");
     }
-    if (!swallowArg)
-      toolArgs += " " + arg;
+    if (!swallowArg) {
+      // salinas toolArgs += " " + arg;
```
Comment (emankov) [inline src/hipBin_amd.h:723]: excessive comment

Comment (david-salinas) [reply src/hipBin_amd.h:723]: opps

File: src/hipBin_amd.h (hunk 1 above)
Comment (david-salinas) [reply src/hipBin_amd.h:226]: good catch.  compilerPath is set by either "getRoccmPath()" or "getHipPath()", which just gets the env var.  Which may, or may not have a trailing slash.

**Pull request closed without merging**